galsim/config/gsobject_ring.py

```diff
@@ -44,7 +45,7 @@ def _BuildRing(config, base, ignore, gsparams, logger):
         full_rotation = math.pi * galsim.radians
 
     dtheta = full_rotation / num
-    if logger and logger.isEnabledFor(logging.DEBUG):
+    if False:
```

Comment:
Not sure how much you care about leaving `if False:` statements in; just wanted to make sure it wasn't accidental.